viz.yaml

```diff
@@ -91,12 +98,26 @@ publish:
     id: text-section
     template: printall
     publisher: section
-    context: 
-      text: 
+    context:
+      text:
         This is a text section in the viz.yaml.  Since it is only
         text, it doesn't need be referenced anywhere outside of the publish
         section.  Text wrapping in the viz should be handled automatically.
-        Follow YAML conventions to change formatting.  
+        Follow YAML conventions to change formatting.
+  -
+    id: markdown-section1
+    template: printall
+    publisher: section
+    depends: markdown-text
+    context:
+      text: markdown-text.example-markdown
```

Comment:
this looks snazzy, but i don't really get it. how does context:text know it needs to parse markdown-text rather than just treating the text as plain text, as in the `text-section` item just above?

Reply:
This is where this gets a bit complicated, in the context building code, it will recognize that `markdown-text` exists and pull the `example-markdown` element from that.  This feature was added mostly to support markdown in a siteText.yaml file that is used to populate the text for a section.

Reply:
can you mix plain text and references to markdown-text?

Reply:
any chance it would work to use context:file instead of context:text or something?

Reply:
it isn't really a file, but an id that references a file, but we don't have any way of specifying that currently